www-client/epiphany/epiphany-44.0.ebuild

```diff
@@ -0,0 +1,86 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit gnome.org gnome2-utils meson xdg virtualx
+
+DESCRIPTION="GNOME webbrowser based on Webkit"
+HOMEPAGE="https://wiki.gnome.org/Apps/Web https://gitlab.gnome.org/GNOME/epiphany"
+
+LICENSE="GPL-3+"
+SLOT="0"
+IUSE="test"
+RESTRICT="!test? ( test )"
+KEYWORDS="~amd64 ~arm ~arm64 ~ppc ~ppc64 ~riscv ~x86"
+
+DEPEND="
+	>=x11-libs/cairo-1.2
+	>=app-crypt/gcr-3.9.0:4=[gtk]
+	>=x11-libs/gdk-pixbuf-2.36.5:2
+	>=dev-libs/glib-2.70.0:2
+	gnome-base/gsettings-desktop-schemas
+	>=media-libs/gstreamer-1.0
```

Comment:
The dependency is `dependency('gstreamer-1.0')`. This usually translates as `media-libs/gstreamer:1.0` with the SLOT specified, FWIW. (I've fixed this locally)